### DISCUSSION TITLE
Ability to load output of command into variable

### DISCUSSION THREAD

**Discussion by ssbarnea** [go-task/task#752] [Q&A] [answered]
One very common use case with build tools is to use the output of a command as a variable, like detecting the version of the package and use it later.

```bash
VERSION=$(cat version.txt)
```

If I am correct, right now there is no way to do this with task, and I might need to use the VERSION variable in multiple tasks.

**Comment by ghostsquad** [comment, marked as answer]
```yaml
VERSION:
  sh: cat version.txt
```
